packages/chrome/src/elements/sheet/components/Close.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { forwardRef, HTMLAttributes } from 'react';
+import XStrokeIcon from '@zendeskgarden/svg-icons/src/16/x-stroke.svg';
+import { IconButton } from '@zendeskgarden/react-buttons';
+
+import { StyledSheetCloseButtonContainer } from '../../../styled';
+
+export const SheetCloseButton = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>(
+  (props, ref) => {
+    return (
+      <StyledSheetCloseButtonContainer>
+        <IconButton aria-expanded aria-label="Close Sheet" ref={ref} {...(props as any)}>
```

Comment:
Will hard coding `aria-label` in English be problematic for i18n?

Reply:
It could be problematic if it is not documented. Is there a standard Garden way of handling internationalization scenarios like this?

Reply:
Looks like the other `Close` buttons in Garden pass through the attribute so that consumers can pass in arbitrary strings. E.g. `<Sheet.Close aria-label={translation} />`. I think the hard-coded `aria-label` can be removed since it prevents users from passing in custom strings, particularly for i18n.

Reply:
According to ARIA the usage for `aria-expanded`: 

> Indicates whether the element, or another grouping element it controls, is currently expanded or collapsed.

My interpretation is that `aria-expanded` is applied to the trigger that controls the sheet — not a close trigger within the sheet. For example, if there was a element that toggles the sheet:

```jsx
<button aria-expanded={isOpen}>Toggle Sheet</button>

<Sheet />
```

Reply:
I do recall that the [Tag close button](https://github.com/zendeskgarden/react-components/blob/main/packages/tags/src/styled/StyledClose.ts#L16) did use an `aria-close` attribute which inclined me to add it. The other reason was to have a default value for `aria-close`. Wouldn't spreading the props after `aria-close` has been defined allow it to be overridden?

Good call on `aria-expanded` I will update this.

Reply:
Removed `aria-expanded`!